pallets/msa/src/lib.rs

```diff
@@ -1,35 +1,13 @@
-//! # MSA Pallet
-//! The MSA pallet provides functionality for handling Message Source Accounts.
+//! Message Source Account Management
```

Comment:
This is here as it is what is picked up by rust docs to show as the short description in the docs search